.gitignore

```
@@ -1,6 +1,6 @@
 # Prevent accidentally committing config file
 # (contains secret bot token)
-src/config.js
+src/instance/config.js
```

Now that we have an instance directory, we can completely ignore the directory and just exclude the example from being ignored. That way we don't need to list every data file we might add in the future
```suggestion
instance/*
!instance/exampleConfig.js
```